lib/msf/core/exploit/vbsobfuscate.rb

```diff
@@ -0,0 +1,40 @@
+# -*- coding: binary -*-
+
+require 'rex/exploitation/vbsobfuscate'
+
+module Msf
+  # VBS obfuscation library wrapper for Rex::Exploitation::VBSObfuscate
+  module Exploit::VBSObfuscate
+    def initialize(info = {})
+      super
+      register_advanced_options([
+        OptInt.new('VbsObfuscate', [false, 'Number of times to obfuscate VBS', 1]),
+      ], Exploit::VBSObfuscate)
```

Comment:
Not a blocker; I don't think this is required anymore 🤔 

```suggestion
      ])
```